wheatley/bot.py

```diff
@@ -71,14 +84,46 @@ def stage(self):
         return self._tower.number_of_bells
 
     # Callbacks
+    def _on_setting_change(self, key, value):
+        def log_warning(message):
```

Comment:
```suggestion
        def log_invalid_key(message):
```

Reply:
Seems like a good plan - I'll fix that now.